front/components/assistant/conversation/AgentSuggestion.tsx

```diff
@@ -97,6 +99,10 @@ export function AgentSuggestion({
       const data = await mRes.json();
       window.alert(`Error adding mention to message: ${data.error.message}`);
     }
+
```

Comment:
Is the call above (edit) awaiting the user message? If not that may have no effect no?